Feat: A user should be able to view the number of todo lists

Qadriyah opened the pull request:
This commit delivers a page where users can view the numbers of
to-do lists they haave ever created and some personal details.
It has adds a navigation menu to Feat: Baker should be able to
create a todo list and add items
[Delivers #160538059], [Delivers #160478650]
@King-Benx
@BarnaTB
@bekeplar
@AtamaZack
@D-Raphael

Qadriyah committed:
Feat: A user should be able to view the number of todo lists

This commit delivers a page where users can view the numbers of
to-do lists they haave ever created and some personal details.
It has adds a navigation menu to Feat: Baker should be able to
create a todo list and add items
[Delivers #160538059], [Delivers #160478650]

BarnaTB reviewed (commented):
I can't seem to find any code from you repo when I clone it to my machine.

Qadriyah commented:
The repository does not have content on the master branch, which is the branch git checks out by default. Be sure to checkout the develop branch and clone again

bekeplar commented:
Make your page more fancy....and also select the best file to retain if u see conflicts

BarnaTB reviewed (changes requested):
Great Work. Though your logout button is broken. I think it'd be better if it linked back to the login page.

Also, your develop branch does not have the changes you say it does. It only has three pages(Login, registration and todo list page)

Qadriyah committed:
Action on feedback

This commit address the following issues:-
1. Use of media queries to make the pages fully responsive
2. Improvements on the color schemes
3. Provision for logging out of the application
4. Improvemnt on the font awesome icons
5. Configured PivotalTracker for public access
[Delivers #160453064], [Delivers #160472200], [Delivers #160478650]

Qadriyah commented:
Thanks Barnabas, the changes will take effect after merging the  feat-profile-page-160538059 branch to the develop. All changes are in that branch

Qadriyah committed:
Action on feedback

This commit address the following isuues:-
1. Linked the logout button back to the login page
2. Submits the login form and redirects to the main dashboard
[Delivers #160478650], [Delivers #160472200], [Delivers #160538059]

D-Raphael commented:
Can't seem to find an index page in your files.

On Wed, 19 Sep 2018 at 00:11, Qadriyah <notifications@github.com> wrote:

> Merged #4 <https://github.com/Qadriyah/Todoapp/pull/4> into develop.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/Qadriyah/Todoapp/pull/4#event-1853509375>, or mute
> the thread
> <https://github.com/notifications/unsubscribe-auth/AYS9Z50tPgR_S84hgUC1OyWakIGrgb6wks5ucWFugaJpZM4Wrl1R>
> .
>


-- 
*M. S. R. Dardai*
dshuuba@outlook.com
dshuuba@gmail.com
*+256786998936*
*+256705771697*

Qadriyah commented:
Actually i was thinking of coming up with a landing page as my index file, but i will make the login page the index file.
